redfish-core/lib/sensors.hpp

```diff
@@ -2909,31 +2909,34 @@ inline void handleSensorGet(App& app, const crow::Request& req,
 
     BMCWEB_LOG_DEBUG << "Sensor doGet enter";
 
-    const std::array<const char*, 1> interfaces = {
+    std::array<const char*, 1> interfaces = {
         "xyz.openbmc_project.Sensor.Value"};
     std::string sensorPath = "/xyz/openbmc_project/sensors/" + nameType.first +
                              '/' + nameType.second;
     // Get a list of all of the sensors that implement Sensor.Value
     // and get the path and service name associated with the sensor
-    crow::connections::systemBus->async_method_call(
-        [asyncResp,
-         sensorPath](const boost::system::error_code ec,
+    ::dbus::utility::getDbusObject(
```

Comment:
can you remove the :: ?

Reply:
It turns out that "::" is needed because the code block is under a namespace

```
2777
2778 namespace sensors   
2779 {
2780
....
2916     // Get a list of all of the sensors that implement Sensor.Value
2917     // and get the path and service name associated with the sensor
2918     ::dbus::utility::getDbusObject(
2919         sensorPath, interfaces,
2920         [asyncResp, sensorId,
2921          sensorPath](const boost::system::error_code& ec,
2922                      const ::dbus::utility::MapperGetObject& subtree) {
```

So without it, it will be treated as `redfish::sensors::dbus::utility::getDbusObject()`.

Here is the compiler log if :: is not used

```
In file included from ../test/redfish-core/lib/sensors_test.cpp:1:
../redfish-core/lib/sensors.hpp: In function ‘void redfish::sensors::handleSensorGet(App&, const crow::Request&, const std::shared_ptr<bmcweb::AsyncResp>&, const std::string&, const std::string&)’:
../redfish-core/lib/sensors.hpp:2918:11: error: ‘redfish::sensors::dbus::utility’ has not been declared
 2918 |     dbus::utility::getDbusObject(
      |           ^~~~~~~

```